Tasks/AzureCloudPowerShellDeployment/Utility.ps1

```diff
@@ -109,7 +121,18 @@ function Get-DiagnosticsExtensions($storageAccount, $extensionsPath, $storageAcc
                         {
                             try
                             {
-                                $publicConfigStorageKey = Get-AzureStorageKey -StorageAccountName $publicConfigStorageAccountName
+                                $publicConfigStorageKey = Get-AzureStorageKey -StorageAccountName $publicConfigStorageAccountName -ErrorAction Ignore
+                                if ($publicConfigStorageKey -eq $null)
+                                {
+                                    #if the storage account is not the classic one, get the primary key in another way
+                                    $publicConfigStorageKey = (Get-AzureRmStorageAccountKey -ResourceGroupName "$storageAccountResourceGroup" -Name "$publicConfigStorageAccountName").Value[0]
```

Comment:
We can only select the classic subscription from VSTS UI for the task "AzurePowerShell", however, "AzurePowerShell" task supports both classic cmdlet and ARM cmdlet.  Is there anything special we need to do for the task "AzureCloudPowerShellDeployment"?